### DISCUSSION TITLE
Why does (Why Should) gap infill ignore overhang speed?

### DISCUSSION THREAD

**Discussion by Aurasphere-au** [OrcaSlicer/OrcaSlicer#6752] [General]
I posted on Bambu/Studio but was suggested to post here and I  think its time I just committed to orca

So I have a situation where the top of the model has an overhang. It works as a vase but when I have layers that then apply gap infill, the inner and outer walls (3 walls) are fine and slow down but the gap infill just flies off and doesnt do what I thought would be reasonable. On the inside it looks pretty messy.

Its a cavity mold insert

Its a question, not of how to do it, as I could just add a modifier and control the gap infill speed on the focus area. Its a workflow question and if gap infill is in an overhang zone then it should respect the speed. These are production parts and must print, lots of them, in one piece, as quickly as possible with as many features managed and no handling afterwards as they fit into production casting after print. Im still having to clean the threads on the bosses as regardless of what I do, the overprint when doing threads has also not been solved.

We never use supports…well unless 101% neccessary.

1. Im checking whether anyone thinks that gap infills during the slow down of an overhang should be related logically
2. Im only able to post the analytics of the model as its client IP under NDA
3. Its a cavity mold so it can’t have any geometry added to the internal space
4. I have pretty good success with convex circular shapes with overhangs and with tapers, its quite easy to do a lot better than with linear segments ie think catenary domes and compression

Im asking the question as it seems like a good place to put an option on gap infill
ie a check box with “Gap Infill Speed Follows Overhangs”

The 3 different components
![image](https://github.com/user-attachments/assets/9fb9a4a2-dc24-4c84-a191-88ce27b92e06)

The speed ranges
![image](https://github.com/user-attachments/assets/9153e871-7b2e-440c-8c6c-463be1643dac)

In other words, if a gap infill cohabitates in a space where there is overhang speed control applied, the speed is higher priority in the time domain so therefore I would have thought that gap infill should follow.
